src/projectscene/view/tracksitemsview/clipcontextmenumodel.cpp

```
@@ -41,18 +41,38 @@ void ClipContextMenuModel::load()
 
     auto colorItems = makeClipColourItems();
 
+    MenuItemList cutAndItems {
+        makeMenuItem("cut-leave-gap",
+                     muse::TranslatableString("clip", "Cut and leave gap")),
```

Cppcheck (2.20.0)

[style] The function 'itemId' is never used.

(unusedFunction)